modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -0,0 +1,43 @@
+= Share a Mule Application as a Template on Exchange
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can publish a Mule application as a template on Exchange, which makes it easy for you and other people to create similar Mule applications in the future.
```

Comment:
...You can share your Mule application with others by publishing it as a template on Exchange... or something similar.

I would use share in the first description here since the title of this section says _share_ and I got confused about _publish_ in the description.
After I continued reading the benefits I understood that by publishing you can share the app. Just a minor change

Reply:
Done.